presto-main/src/main/java/io/prestosql/sql/planner/planprinter/TextRenderer.java

```diff
@@ -254,6 +249,14 @@ private String printEstimates(PlanRepresentation plan, NodeRepresentation node)
         return output.toString();
     }
 
+    private static boolean hasChildren(NodeRepresentation node, PlanRepresentation plan)
+    {
+        return node.getChildren().stream()
+                .map(plan::getNode)
+                .filter(Optional::isPresent)
```

Comment:
```suggestion
                .anyMatch(Optional::isPresent);
```

Reply:
Ah, good catch. I just extracted a method from preexisting code. I'll fix it.